source/go-perun/app_tutorial/test.rst

```diff
@@ -0,0 +1,241 @@
+Test
+====
+We now create a simple test for our app channel client by setting up two clients, opening an app channel between them, and playing one round of Tic-Tac-Toe.
+Everything in this section will take place in package ``main``.
+
+.. code-block:: go
+
+    package main
+
+Setup
+-----
+
+We construct helpers in ``util.go`` for contract deployment, client generation, and logging of balances.
+
+.. note::
+    These are very similar to the previous tutorial; hence only the additions in ``deployContracts`` and ``setupGameClient`` are mentioned here.
+
+    **Deploy contracts.**
+    Besides the Adjudicator and Asset Holder, the ``TicTacToeApp.sol`` app contract is required to be deployed.
+    Its go binding provides ``ticTacToeApp.DeployTicTacToeApp`` to realize this.
+
+    **Setup game client.**
+    The ``app`` and ``stake`` arguments are added here to satisfy the client's constructor ``client.SetupAppClient``.
```

Comment:
Might actually be fine to not show the new code here, but links to the corresponding parts in the source code would still be nice to have.